youtrack_sdk/client.py

```diff
@@ -261,6 +261,21 @@ def update_issue_comment(self, *, issue_id: str, comment: IssueComment) -> Issue
             ),
         )
 
+    def update_issue(self, *, issue_id: str, issue: Issue) -> Issue:
```

Comment:
Could you please move `update_issue` after `create_issue` method?

Reply:
Done!